### PR TITLE
Add RC channel usage to config page

### PR DESCRIPTION
**Pull request by 1Blademaster**
image

### CODE REVIEW

File: radio/app/controllers/rcController.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            ).get("data")
+
+            if min_param:
+                channel_params["min"] = min_param.param_value
+            if max_param:
+                channel_params["max"] = max_param.param_value
+            if reversed_param:
+                channel_params["reversed"] = reversed_param.param_value
+            if option_param:
+                channel_params["option"] = option_param.param_value
```
Comment (NexInfinite) [inline radio/app/controllers/rcController.py:61]: code is very duplicated, why not a function that takes in the param name (like `RC{channel_number}_MIN`) and does the `self.drone.paramsController.get...` and then `if xxxx: channel_params[....`?

**Review by NexInfinite — CHANGES_REQUESTED** [review]
Love the tests, could improve on the code dup

File: radio/app/controllers/rcController.py (hunk 1 above)
Comment (1Blademaster) [reply radio/app/controllers/rcController.py:61]: We still need to check if the param exists, we don't want to add it if it's None so I don't think adding another function does much

Comment (1Blademaster) [reply radio/app/controllers/rcController.py:61]: But that's only for the `channel_params`, right? I'd have to make a second one for the `self.params`

**Review by NexInfinite — APPROVED** [review]
LGTM

**Pull request merged**